bindings/csharp/Regorus/Regorus.cs

```diff
@@ -0,0 +1,305 @@
+using System.Runtime.InteropServices;
+using System.Text;
+using System;
+
+#nullable enable
+namespace Regorus
+{
+	public sealed class Exception : System.Exception
```

Comment:
I would either rename this, or just use the built-in `System.Exception`. Otherwise when consumers import the `Regorus` namespace, they may run into naming conflicts if they reference the existing `Exception` class.